ros/src/models/chrono/ros_chrono/src/path_following.cpp

```diff
@@ -115,14 +110,14 @@ double fps = 60;
 int filter_window_size = 20;
```

Comment:
what is this? Can we move it to the YAML file as well in case it changes?

Reply:
This part of codes comes from hmmwv_model.cpp (link: https://github.com/ChangYuHsuan/MAVs/blob/master/ros/src/models/chrono/ros_chrono/src/hmmwv_model.cpp ), which is written in last semeter. We didn't change the logic of this code too much. Actually, all the cpps, including hmmwv_model.cpp, steering_controller.cpp, path_following.cpp, trajectory_follower.cpp, all have the same part of code here. Do you want to fix all of the cpps?

Reply:
Okay